js/5etools-backgrounds.js

```diff
@@ -115,16 +117,49 @@ function d20plusBackgrounds () {
 
 		const renderer = new Renderer();
 		renderer.setBaseUrl(BASE_SITE_URL);
-		const renderStack = [];
-		let feature = {};
+		const featureSet = [
+			"Cultural Chameleon",
```

Comment:
this is not maintainable (and doesn't support homebrew, besides)--the data* already has `.data.isFeature = true` for features; this should be used instead of matching by name

\* the 5etools source data, at any rate; if  that doesn't make its way into here, then we should fix that first

Reply:
Explicitly tagged the entries as features